src/main/webapp/dotenv.js

```diff
@@ -9,10 +9,5 @@ window.env = {
 	ENDPOINT_URL: "https://analytics.opendatahub.testingmachine.eu",
 	ODH_MOBILITY_API_URI: "https://mobility.api.opendatahub.testingmachine.eu",
 	LOG_APPLICATION_NAME: "opendatahub-analytics",
-	THUNDERFOREST_MAP_API_KEY: "",
-	KEYCLOAK_AUTHORIZATION_URI: "https://auth.opendatahub.testingmachine.eu/auth",
-	KEYCLOAK_REALM: "noi",
-	KEYCLOAK_CLIENT_ID: "odh-mobility-analytics",
-	KEYCLOAK_REDIRECT_URI: "http://localhost:8999/",
-	KEYCLOAK_SILENT_CHECK_SSO_REDIRECT_URI: "http://localhost:8999/callback.html",
```

Comment:
Hi @noctho would it be a big problem to keep these values here. Such that, we have the whole configuration of the webapp in a single place?

Reply:
Of course we can keep the old configuration. I only changed it because in the sample project the keycloak configuration was in the keycloak.json file.

Reply:
@noctho Yes, sorry for the inconvenience. The example project did not consider the single-point-of-configuration policy...